File: app/src/main/java/com/draabek/fractal/canvas/CpuFractal.java
package com.draabek.fractal.canvas;

import com.draabek.fractal.fractal.Fractal;
import com.draabek.fractal.fractal.FractalViewWrapper;

/**
 * Created by Vojta on 07.09.2017.
 */

public class CpuFractal extends Fractal {

    @Override
    public Class<? extends FractalViewWrapper> getViewWrapper() {
        return FractalCpuView.class;
    }
}

File: app/src/main/java/com/draabek/fractal/canvas/CpuFractal.kt
package com.draabek.fractal.canvas

import com.draabek.fractal.fractal.Fractal
import com.draabek.fractal.fractal.FractalViewWrapper

open class CpuFractal : Fractal() {
    override val viewWrapper: Class<out FractalViewWrapper?>
        get() = FractalCpuView::class.java
}
